src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -141,7 +141,8 @@ private void createTemplateFeedbackQuestions(String courseId, String feedbackSes
         return new ArrayList<FeedbackQuestionAttributes>();
     }
 
-    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+    @Override
+    protected FeedbackSessionAttributes extractFeedbackSessionDataSetUniqueAttributes(
```

Comment:
Yeah but if i just give  `SetUniqueAttributes` wouldn't it be difficult in sub classes to make out what's happening ?.